src/utils/exponential-backoff.js

```diff
@@ -33,28 +33,28 @@ const EXPONENTIAL_BACKOFF_DELAY = 5000
  * @param {Function} callFunction
  * @param {Object} [payload={}] - information for error log during backoff failures
  * @param {Object} opts
- * @param {String} opts.errorMessage
+ * @param {String} opts.debugName
  * @param {Number} [attempts=EXPONENTIAL_BACKOFF_ATTEMPTS] opts.attempts - attempts left
  * @param {Number} [delayTime=EXPONENTIAL_BACKOFF_DELAY] opts.delayTime - delay in milliseconds between calls
  * @param {Logger} [delayTime=EXPONENTIAL_BACKOFF_DELAY] opts.logger
  * @return {Promise}
  */
-async function exponentialBackoff (callFunction, payload = {}, { errorMessage = null, attempts = EXPONENTIAL_BACKOFF_ATTEMPTS, delayTime = EXPONENTIAL_BACKOFF_DELAY, logger = console }) {
+async function exponentialBackoff (callFunction, payload = {}, { debugName = null, attempts = EXPONENTIAL_BACKOFF_ATTEMPTS, delayTime = EXPONENTIAL_BACKOFF_DELAY, logger = console }) {
   try {
     var res = await callFunction()
   } catch (error) {
     if (attempts > 0) {
       const attemptsLeft = attempts - 1
       const nextDelayTime = delayTime * DELAY_MULTIPLIER
 
-      if (errorMessage) {
```

Comment:
Why remove this functionality?

Reply:
@treygriffith that was the only place we used it (and didnt need it)